### PR TITLE
Create Navbar, login & Signup

### PR DESCRIPTION
**Pull request by adi061001**
create Navbar, login & Signup with Routing

### CODE REVIEW

File: client/src/component/button.tsx
Comment (navneets0396) [inline client/src/component/button.tsx:1]: Change the name to Button

File: client/src/Screen/Login/LoginPage.tsx
Comment (navneets0396) [inline client/src/Screen/Login/LoginPage.tsx:1]: Either use sx with MUI component or the best way you can use styled component, avoid inline styling.

File: client/src/component/button.tsx
Comment (navneets0396) [inline client/src/component/button.tsx:1]: Change the name to Button

File: client/src/Screen/Login/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                  />
+                  <br></br>
+                  <Input
+                    type="password"
+                    label="Password"
+                    style={{ mt: 1, width: 300 }}
+                    emailid={inputLogin}
+                    name="loginPassword"
+                  />
+                  <br></br>
```
Comment (navneets0396) [inline client/src/Screen/Login/LoginPage.tsx:145]: The use of br tag is wrong check the syntax and better go with box when using MUI.

File: client/src/Screen/Login/LoginPage.tsx
Comment (adi061001) [reply client/src/Screen/Login/LoginPage.tsx:1]: ohk sir ,but i am using sx where i have use style  name as props for reusable component

**Pull request closed without merging**